packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -613,12 +613,12 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
   @protected
   double scrollOffsetOf(RenderSliver child, double scrollOffsetWithinChild);
 
-  /// Returns the total scroll obstruction extent of all slivers in theild].
```

Comment:
Did you double check to make sure this is right?

Reply:
Yes, I spoke with Ian about this one.